output/theme.sasslint.txt

```diff
@@ -22,11 +22,14 @@ src/theme/_forms.scss
   337:9   error    Qualifying elements are not allowed for class selectors  no-qualifying-elements
   417:8   warning  Vendor prefixes should not be used                       no-vendor-prefixes
 
+src/theme/_navbar.scss
+  18:6  warning  Expected indentation of 4 tabs but found 5  indentation
```

Comment:
except that all fine

Reply:
Looks like prettier is moving it :thinking: